instrumentation/nacos-client-2.0.3/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.alibaba.nacos")
+    module.set("nacos-client")
+    versions.set("[2.0.3,)")
+    skip("0.5.0", "0.6.1", "1.1.2", "1.1.4", "1.4.7", "2.0.1", "2.0.2")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  val nacosClientVersion = "2.0.3"
```

Comment:
The version 2.0.3 was chosen because I referred to the requirements described in issue #9961. However, after testing, I found that version 2.0.0 also works. I will make changes here.